public/video-ui/src/services/VideosApi.js

```diff
@@ -168,24 +168,32 @@ export default {
     });
   },
 
-  createComposerPage(id, metadata, composerUrlBase) {
+  createComposerPage(id, video, composerUrlBase) {
+
+    const composerData = getComposerData(video);
+
     const initialComposerUrl =
       composerUrlBase +
       '/api/content?atomPoweredVideo=true&originatingSystem=composer&type=video';
 
-    const properties = composerSyncFields.reduce((queryStrings, property) => {
-      if (metadata[property]) {
+    const properties = composerData.reduce((queryStrings, data) => {
+      if (data.value) {
         queryStrings.push(
           '&initial' +
-            property.charAt(0).toUpperCase() +
-            property.slice(1) +
+            data.name.charAt(0).toUpperCase() +
+            data.name.slice(1) +
             '=' +
-            metadata[property]
+            data.value
         );
       }
       return queryStrings;
     }, []);
-    const composerUrl = initialComposerUrl + properties.join('');
+
+    let composerUrl = initialComposerUrl + properties.join('');
```

Comment:
I thought about that it but it ended up being quite a long statement and I thought this one was clearer.